app/reports/hbx_reports/shop_enrollment_report.rb

```diff
@@ -94,14 +94,15 @@ def writing_on_csv(enrollment_ids_final, glue_list)
             enrollees_hbx_ids = hbx_enrollment.hbx_enrollment_members.map(&:hbx_id).join(', ')
             enrollees_dob = hbx_enrollment.hbx_enrollment_members.map { |member| member.person.dob }.join(', ')
             osse_eligible = (hbx_enrollment.eligible_child_care_subsidy > 0 ? "Yes" : "No")
+            total_responsible_amount = BigDecimal((calculator.total_premium.to_f - calculator.total_employer_contribution.to_f - hbx_enrollment.eligible_child_care_subsidy.to_f).to_s).round(2)
```

Comment:
should we wrap these in parentheses? I am not sure the order of execution intended. 

<img width="192" alt="Screen Shot 2023-04-19 at 8 52 05 AM" src="https://user-images.githubusercontent.com/118293805/233096570-44f45762-807a-4d8f-8047-434b6ab64359.png">

Reply:
this is good to have for readability and for people looking into the code for the first time.